source/installation/install-on-an-unattended-server/windows-server-installation/windows-external-access.txt

```diff
@@ -12,8 +12,8 @@ Enable External Access to Relational Migrator on Windows
    :depth: 1
    :class: singlecol
 
-Enable external access to Relational Migrator on Windows. By enabling external 
-access, incoming connections are allowed on the same port as Relational Migrator. 
+Enable external access to Relational Migrator on Windows. Enabling external 
+access allows incoming connections on the same port as Relational Migrator. 
 
```

Comment:
What if you deleted the first sentence all together? I think it is a bit redundant. Then, with the second sentence, 

"To allow incoming connections on the same port as Relational Migrator, enable external access to ..."